T2CC_APP/andriod app/COSC612StudentAndriod/app/src/main/res/layout-land/activity_main.xml

```diff
@@ -0,0 +1,84 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
it'd be nice if we could put in similar icons from the GUI design diagram. Might need to ask Taps for those.

Could we also rename this file to be activity_login. This is not a big deal but it makes things clearer